### PR TITLE
Replace webmozart/path-util with symfony/filesystem

### PR DESCRIPTION
**Pull request by mamazu**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,8 +10,7 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
-        "psr/log": "^1.0",
-        "webmozart/path-util": "^2.3"
+        "psr/log": "^1.0"
```
Comment (dantleech) [inline composer.json:13]: needs to be in `require`

Comment (mamazu) [reply composer.json:13]: I have just removed it. Should I also move the logger to the dev dependencies?

**Comment by mamazu** [comment]
And about the composer plugins I don't know how we want to handle those.

File: composer.json (hunk 1 above)
Comment (dantleech) [reply composer.json:13]: sorry, i meant, symfony/filesystem needs to be in `require`, assuming that this library used the `Path::` function

**Comment by dantleech** [comment]
r.e. the composer plguins, just do what composer says and allow them in `composer.json`

File: composer.json (hunk 1 above)
Comment (mamazu) [reply composer.json:13]: Ah makes sense, I'll do that.

**Comment by mamazu** [comment]
Done

**Comment by mamazu** [comment]
Is there anything else left to do?

**Comment by dantleech** [comment]
thanks

**Pull request merged**